change/@internal-react-composites-ab127372-d7f5-49a4-b1ae-fa277b1b836e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "added proper Icon",
```

Comment:
This comment should make sense standalone in a CHANGELOG file. How about "Update CallComposite icons to include camera cycler button icon"